packages/hooks/src/use_resource.rs

```diff
@@ -11,8 +11,8 @@ use futures_util::{future, pin_mut, FutureExt, StreamExt};
 use std::ops::Deref;
 use std::{cell::Cell, future::Future, rc::Rc};
 
-/// A memo that resolve to a value asynchronously.
-/// Unlike `use_future`, `use_resource` runs on the **server**
+/// A memo that resolves to a value asynchronously.
+/// Similar to `use_memo` but `use_resource` runs asynchronously.
```

Comment:
`use_resource` will not memorize the value returned from the resource unlike `use_memo`. It is pretty similar to `use_future` with a return value